src/channel-slider/channel-slider.js

```diff
@@ -178,7 +192,8 @@ export default class ChannelSlider extends NudeElement {
 		defaultValue: {
 			type: Number,
 			default () {
-				return this.defaultColor.get(this.channel);
+				let color = this.defaultColor;
+				return this.channel === "alpha" ? color.alpha : color.get(this.channel);
```

Comment:
Does `color.get("alpha")` not work? I think that's a reported Color.js bug, it may be worth it to fix it there